db/memorystore.go

```diff
@@ -16,6 +16,17 @@ import (
 	"github.com/letsencrypt/pebble/core"
 )
 
+// Error when key change fails because new account key has already been in use
+type ExistingAccountError struct {
+	msg             string
+	MatchingAccount *core.Account
+}
+
+func (e *ExistingAccountError) Error() string {
+	return e.msg
```

Comment:
I think you can remove the `msg` field from this struct entirely since the only place `ExistingAccountError` is used is for a new key conflict and the message for that case is always the same. If you remove `msg` you can implement `Error()` with: 
`return fmt.Sprintf("New key is already in use by account %s", e.ID)`

Reply:
Done.